apps/client/src/main.tsx

```diff
@@ -12,7 +12,20 @@ const root = ReactDOM.createRoot(
 root.render(
   <StrictMode>
     <Provider store={store}>
-      <App />
+      <BrowserRouter>
+        <Routes>
+          <Route path='/' element={<Layout />}>
+            <Route index element={<Home />} />
+            <Route path=':id'>
+              <Route index element={<Home />} />
+              <Route path='room' element={<Room />} />
+              <Route path='race' element={<Race />} />
+              <Route path='leaderboard' element={<Leaderboard />} />
+            </Route>
+          </Route>
+          <Route path='*' element={<Navigate to='/' />} />
```

Comment:
Better to add a 404 page from start

Reply:
Added 0552de4faf3c4060bf3af438fa8cb0eb6467254b